tests/profile.spec.js

```diff
@@ -52,11 +52,25 @@ test("Profile not found redirects to search page with error message", async ({
   );
 });
 
-test.fixme("Link navigates", async () => {
-  // 1. navigate to profile
-  // 2. get a link and href
-  // 3. click the link
```

Comment:
I think a step is missing, clicking of the link to make sure it navigates to the correct location